l10n_mapper_generator/lib/l10n_mapper_generator.dart

```diff
@@ -98,6 +107,9 @@ class L10nMapperGenerator extends Generator {
           // skips gen-exceptions
           if (genExceptions.contains(name)) continue;
 
+          // skip fields not in mapperWhitelist
+          if (mapperWhitelist.isNotEmpty && !mapperWhitelist.contains(name)) continue;
```

Comment:
There is no need specifying a separate `mapperWhitelist` and a check for this. See `genExceptions` variable in  `l10n_mapper_generator.dart` file. 

You just have to add these parsed exceptions to the existing list on instantiating `L10nMapperGenerator`.

Reply:
Currently `l10n_mapper` package already applies the concept of `excepting` some keys statically defined in `genExceptions` object and my point is to utilise this instead of the need to use an extra check for similar usecase.

Example:
```dart
  ...
  final List<String> _exceptions;

  L10nMapperGenerator({
    required this.l10n,
    required this.locale,
    required this.parseL10n,
    required this.message,
    required List<String> exceptions,
  }): _exceptions = [...genExceptions, ...exceptions];

// use `_exceptions` were applicable
if (_exceptions.contains(name)) continue;
```